src/librbd/ImageState.h

```diff
@@ -26,7 +26,9 @@ class ImageState {
   ~ImageState();
 
   int open();
+  int open(bool skip_open_parent);
```

Comment:
Nit: can you just combine these methods and fix the callsites that now need to pass "false"?